docs/drivers/azure.md

```diff
@@ -69,6 +69,7 @@ Optional:
 - `--azure-docker-port`: Port number for Docker engine.
 - `--azure-environment`: Azure environment (e.g. `AzurePublicCloud`, `AzureChinaCloud`).
 - `--azure-storage-type`: Type of Azure Storage account hosting the OS disk of the machine (e.g. `Standard_LRS`, `Premium_LRS`).
+- `--azure-custom-data`: Path to the CustomData file.
```

Comment:
CLI and docs do not match in `CustomData` vs `custom-data`. We often have more info in the markdown, so perhaps just say `Path to the custom-data file (i.e. cloud-init/cloud-config scripts).`